docs/tutorials/fundamentals/installation.md

```diff
@@ -172,8 +173,9 @@ scientific packages such as Spyder or matplotlib. If neither is available,
 running napari will result in an error message asking you to install one of
 them.
 
-Running `pip install "napari[all]"` will install the default framework – currently
-PyQt5, but this could change in the future.
+Running `pip install "napari[all]"` will install the default Qt framework, which is currently 
+PyQt5--but this could change in the future. (If you have a Mac with arm64 architecture (Apple
+Silicon), this will not work--see {ref}`note-m1`.)
```

Comment:
```suggestion
PyQt5--but this could change in the future. If you have a Mac with a newer arm64 architecture (Apple
Silicon), this will not work--see {ref}`note-m1`.
```